package/MDAnalysis/core/universe.py

```diff
@@ -1498,8 +1498,11 @@ def Merge(*args):
                    residue_segindex=segidx)
 
     # Create and populate a universe
-    coords = np.vstack([a.positions for a in args])
-    u = Universe(top, coords[None, :, :],
+    try:
+        coords = np.vstack([a.positions for a in args])
+        u = Universe(top, coords[None, :, :],
                  format=MDAnalysis.coordinates.memory.MemoryReader)
-
+    except:
```

Comment:
Explicitly list the exceptions on which the except clause triggers. Blanket exception catching will lead to very hard to find bugs.